Provider/src/main/java/com/spotify/confidence/CommonContextKeys.kt

```diff
@@ -0,0 +1,38 @@
+package com.spotify.confidence
+object CommonContextKeys {
```

Comment:
Would be nice to have this in a separate PR, and perhaps be worked on in conjunction with iOS to make sure all of these fields work well in "all" mobile paltforms